app/views/templates/Layout.scala.html

```diff
@@ -14,11 +30,13 @@
     hmrcTimeoutDialog: HmrcTimeoutDialog,
     hmrcReportTechnicalIssueHelper: HmrcReportTechnicalIssueHelper,
     hmrcScripts: HmrcScripts,
-    alphaBanner: StandardAlphaBanner
+    betaBanner: StandardBetaBanner
+
 )
 
 @(pageTitle: String, showBackLink: Boolean = true, timeout: Boolean = true, showSignOut: Boolean = true)(contentBlock: Html)(implicit request: Request[_], messages: Messages)
 
+@import uk.gov.hmrc.hmrcfrontend.views.config.StandardBetaBanner
```

Comment:
maybe move this import up to the others?

Reply:
This is fixed now :)